### PR TITLE
[CNSM-865] Search V2 documentation

### PR DESCRIPTION
**Pull request by abhisheksarka**
Ticket: https://accredible.atlassian.net/browse/CNSM-865
Design: https://accredible.atlassian.net/wiki/spaces/PD/pages/3609526374/New+Credentials+Public+Search+API+using+OS

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -938,6 +938,112 @@ Create bulk Credentials using this action. It requires a JSON object containing
...
+    Supported operations - `eq`, `in`, `lt`, `lte`, `gt`, `gte`
+    + page (object, optional)
+        + from - The pagination page to start from
+        + size - The number of records to be returned. Default is 50
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Token token=YOUR API_KEY
```
Comment (michael-accredible) [inline apiary.apib:988]: Shall we add `Content-Type` as well?

File: apiary.apib [hunk 2]
```diff
@@ -938,6 +938,112 @@ Create bulk Credentials using this action. It requires a JSON object containing
...
+| `eq`      | Exact match (default, if nothing specified). | `{"group.id[eq]": 1}`<br>or<br>`{"group.id": 1}`<br><br>Both will yield the same result. |
+| `in`      | Allows an array of values.                   | `{"group.id[in]": [1, 2]}`                                                               |
+| `lt`      | Less than.                                   | `{"issued_on[lt]": "31/12/2023"}`                                                        |
+| `gt`      | Greater than.                                | `{"issued_on[gt]": "01/01/2023"}`                                                        |
+| `lte`     | Less than equal to.                          | `{"expired_on[lte]": "31/12/2023"}`                                                      |
+| `gte`     | Greater than equal to.                       | `{"expired_on[gte]": "01/01/2023"}`                                                      |
+
+### Search for Credentials [POST]
+
++ Attributes
```
Comment (michael-accredible) [inline apiary.apib:957]: The query payload should be wrapped in a `query` object.
```
{
  query: {
    <queries>
  },
  page: { ... }
}
```

File: apiary.apib (hunk 1 above)
Comment (abhisheksarka) [reply apiary.apib:988]: Line 984 already has it

**Review by michael-accredible — APPROVED** [review]

**Review by marwan-tanager — APPROVED** [review]

File: apiary.apib [hunk 3]
```diff
@@ -938,6 +938,116 @@ Create bulk Credentials using this action. It requires a JSON object containing
...
+    + query (object, optional)
```
Comment (michael-accredible) [inline apiary.apib:958]: This is `required` as per our discussion.

**Review by naveenbalachandar — APPROVED** [review]

**Pull request merged**